geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -1823,6 +1823,8 @@
       "Cannot find regions <{0}> in any of the members";
   public static final String QUERY__MSG__NOT_SUPPORTED_ON_MEMBERS = CliStrings.QUERY
       + " command should be used only from shell. Use QueryService API for running query inside Geode VMs";
+  public static final String QUERY__MEMBER__HELP =
+      "Name/Id of a member which hosts the regions to be queried.";
```

Comment:
Maybe change this to `Name/Id of the member on which to execute the query` instead?, I think is easier to understand and doesn't generate doubts on the operator/user executing the command.

Reply:
+1